@xen-orchestra/lite/src/views/vm/VmConsoleView.vue

```diff
@@ -128,7 +129,12 @@ const toggleFullScreen = () => {
 };
 
 const openInNewTab = () => {
-  const routeData = router.resolve({ query: { ui: "0" } });
+  const externalMaster = getFirst(route.query["master"]);
+  const query: { ui: string; master?: string } = { ui: "0" };
+  if (externalMaster != undefined) {
+    query["master"] = externalMaster;
+  }
+  const routeData = router.resolve({ query });
```

Comment:
I think this should be enough:

```suggestion
  const routeData = router.resolve({ query: { ...route.query, ui: "0" } });
```